.github/workflows/get-runner-access.yaml

```
@@ -0,0 +1,24 @@
+name: golangci-lint
+on:
+  pull_request:
+
+jobs:
+  get-runner-shell:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.4.2
+      with:
+        minikube version: 'v1.23.2'
+        kubernetes version: 'v1.22.2'
+        start args: --memory 6g --cpus=2
+        github token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        github token: ${{ secrets.GITHUB_TOKEN }}
```
we need to add token to this repo with this `GITHUB_TOKEN` name

---

this is the default token we don't need to add it right?

---

oh I mixed with custom variables.